blueprints/app/files/bower.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "<%= name %>",
+  "dependencies": {
+    "ember": "2.0.0",
```

Comment:
As embers version changes, as do these dependencies. (once we drop bower though, that problem gets much nicer)